modules/cluster-backups/outputs.tf

```diff
@@ -1,14 +1,15 @@
 output "s3" {
   value = {
-    id  = data.aws_s3_bucket.this.id
-    arn = data.aws_s3_bucket.this.arn
+    id  = local.bucket_exists ? one(data.aws_s3_bucket.this.*.id) : null
```

Comment:
You can reduce these down to just `id = one(data.aws_s3_bucket.this.*.id)` as `one()` on an empty list will default to null.